src/Avatar/Avatar.js

```diff
@@ -9,19 +9,17 @@ type Props = {
   /** Background color */
   background?:
     | 'blue'
+    | 'bronze'
```

Comment:
can we share this union type with the other places that use it? maybe export the type from the color file?

looks like a few files in PaletteDemo

Reply:
Given palette demo's state of affairs, we opted for as light a touch as reasonable.

Also, the type used in Avatar is not the same as (most of) those in PaletteDemo.